cmd/cmdutil/term.go

```diff
@@ -0,0 +1,37 @@
+package cmdutil
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+func ReadFromTerm(prompt string, maxtries int, valid func(string) bool) (string, error) {
```

Comment:
This function seems to be unused in this PR, so this comment can be skipped. 

Should we add a check for maxtries to be `>= 1` ?

Reply:
Yeah, sorry, this code slipped in from splitting up a huge change into multiple PRs.

I probably should have documented the function a little better, and the variable is also named improperly.

`maxtries` can be -1, in which case the number of tries are infinite. If `maxtries` is 0, only a single attempt is allowed. Otherwise, `maxtries` retries are allowed. Probably, the variable should be named `maxRetries` to reflect what it does.

Let me fix that.

Reply:
Fixed. Is it more clear now?

Reply:
Thanks for the detailed explanation. Comments added look good.